docs/installation/getting-started/predefined-builds.md

```diff
@@ -877,7 +877,17 @@ The table below presents the list of all plugins included in various builds. <!-
 				<td style="text-align:center; width:70px">✅</td>
 			</tr>
 			<tr>
-				<td><a href="https://ckeditor.com/docs/ckeditor5/latest/api/essentials.html">Essentials</a> *</td>
+				<td><a href="https://ckeditor.com/docs/ckeditor5/latest/api/essentials.html">Essentials</a> *<br />
+				Includes:</br />
+				<ul>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/clipboard.html">Clipboard</a></li>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/enter.html">Enter</a></li>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/select-all.html">SelectAll</a></li>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_enter_shiftenter-ShiftEnter.html">ShiftEnter</a></li>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/typing.html">Typing</a></li>
```

Comment:
Nitpick: `Typing` is a glue that pulls `Input` and `Delete`. Should we mention them?

This is true for some other plugins (i.e. `Undo` -> `UndoEditing`, `UndoUI`), but my comment does not apply to them, because I consider split to `Editing` and `UI` as technical detail.

Reply:
We may discuss exploding all such case in the table. it's so large anyway that it will not adversely affect anything.